<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapConfigDataClass</class>
 <widget class="QMainWindow" name="MapConfigDataClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Map Config Data</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Справка</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;MCD&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; (Map Config Dat) - утилита которая встроит в карту метаданные, которые помогут хостботам точнее опознать карту (если они это поддерживают). По своей сути заменяет cfg файл, который обычно лежит рядом с картой.&lt;br/&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; font-style:italic;&quot;&gt;MCD&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;&gt; перепишет некоторые байты, которые карте особо не нужны.&lt;br/&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:12pt; text-decoration: underline;&quot;&gt;Откройте карту, чтобы узнать какие данные можно записать в карту.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Разработано для &lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#55007f;&quot;&gt;IrIna&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; Bot. &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="MFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="MOpen"/>
    <addaction name="MSaveAs"/>
    <addaction name="MSaveMetadata"/>
   </widget>
   <widget class="QMenu" name="MAdd">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Добавить</string>
    </property>
    <addaction name="MAutohost"/>
    <addaction name="action"/>
   </widget>
   <addaction name="MFile"/>
   <addaction name="MAdd"/>
  </widget>
  <action name="MOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="MSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="MSaveMetadata">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Вставить метаданные</string>
   </property>
  </action>
  <action name="MAutohost">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Автохост</string>
   </property>
   <property name="CustomIndex" stdset="0">
    <number>8</number>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Слоты</string>
   </property>
   <property name="CustomIndex" stdset="0">
    <number>2</number>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mapconfigdata.qrc"/>
 </resources>
 <connections/>
</ui>
